website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+
+</detailsToggle>
+
+<detailsToggle alt_header="If I run `dbt build --select +model`, will this trigger a run of upstream models in other projects?">
+
+No, unless downstream projects are installed as [packages](/docs/build/packages) (source code). In that case, the models in project installed as a project become “your” models, and you can select or run them. There are cases in which this can be desirable; see docs on [project dependencies](/docs/collaborate/govern/project-dependencies).
+
+</detailsToggle>
+
+<detailsToggle alt_header="If each project/domain has its own data warehouse, is it still possible to build models across them?">
+
+Yes, as long as they’re in the same data platform (BigQuery, Databricks, Redshift, Snowflake, etc.) and you have configured permissions and sharing in that data platform provider to allow this.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can I run tests that involve tables from multiple different projects?">
+
+Yes, because the cross-project collaboration is done using the `{{ ref() }}` macro, you can use those models from other teams in [singular tests](/docs/build/data-tests#singular-data-tests). 
+
+</detailsToggle>
+
+<detailsToggle alt_header="Which team's data schema would dbt Mesh create?">
+
+Each team defines their connection to the data warehouse, and the default schema names for dbt to use when materializing datasets.
+
+By default, each project belonging to a team will create:
+
+- One schema for production runs (for example, `finance`).
+- One schema per developer (for example, `dev_jerco`).
+
+Depending on each team’s needs, this can be customized with model-level [schema configurations](/docs/build/custom-schemas), including the ability to define different rules by environment.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is it possible to apply model contracts to source data?">
+
+No, contracts can only be applied at the [model level](/docs/collaborate/govern/model-contracts). It is a recommended best practice to [define staging models](/best-practices/how-we-structure/2-staging) on top of sources, and it is possible to define contracts on top of those staging models.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can contracts be partially enforced?">
+
+No. A contract applies to an entire model, including all columns in the model’s output. This is the same set of columns that a consumer would see when viewing the model’s details in Explorer, or when querying the model in the data platform. 
+
+- If you wish to contract only a subset of columns, you can create a separate model (materialized as a view) selecting only that subset. 
+- If you wish to limit which rows or columns a downstream consumer can see when they query the model’s data, depending on who they are, some data platforms offer advanced capabilities around dynamic row-level access and column-level data masking.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can you have multiple owners in a group?">
```

Comment:
```suggestion
<detailsToggle alt_header="Can I have multiple owners in a group?">
```

The questions are generally worded in the 1st person in this way, not 2nd person.